react-native-share-extension.podspec

```diff
@@ -0,0 +1,19 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name         = package['name']
+  s.version      = package['version']
+  s.summary      = package['description']
+  s.license      = package['license']
+
+  s.authors      = package['author']
+  s.homepage     = package['homepage']
+  s.platform     = :ios, "9.0"
+
+  s.source       = { :git => "https://github.com/MattiasMansson/react-native-share-extension", :tag => "v#{s.version}" }
```

Comment:
Do you know what will happen here if you decided to remove your fork?

Reply:
Oh right, I need to update it to match your repository